### PR TITLE
Show time change on datetime header

### PR DESCRIPTION
**Pull request by JCrestel**
Add a readme  

Change validate event on validate button

Show time change on datetime header

### CODE REVIEW

**Comment by Wikiki** [comment]
Integrated in v6.0.8

File: src/js/index.js [hunk 1]
```diff
@@ -317,8 +317,9 @@ export default class bulmaCalendar extends EventEmitter {
     this.timePicker.value(new Date());
     this.timePicker.refresh();
 
-    // TODO: check if closeOnSelect
-    this.save();
+    if (this.options.closeOnSelect && this.options.displayMode !== 'inline') {
+      this.save();
+    }
```
Comment (jonathanpglick) [inline src/js/index.js:322]: Can you explain why this change was made? Moving the `save()` and `emit()` calls inside the conditional prevents the `select` event from firing in most cases.

File: src/js/index.js [hunk 2]
```diff
@@ -262,14 +262,14 @@ export default class bulmaCalendar extends EventEmitter {
...
   onSelectDateTimePicker(e) {
-    this.refresh();
-    this.save();
-    if (e.type === 'select' && this.options.closeOnSelect && this.options.displayMode !== 'inline') {
+    this.refresh(); 
+    if (e.type === 'select' && this.options.closeOnSelect === true && this.options.displayMode !== 'inline') {
+      this.save();
       this.hide();
+      this.emit(e.type, this);
     }
```
Comment (jonathanpglick) [inline src/js/index.js:270]: Can you explain why this change was made? Moving the `save()` and `emit()` calls inside the conditional prevents the `select` event from firing in most cases.
